### ISSUE TITLE
Add buffer to text in example box

### ISSUE THREAD

**Issue by cbahlai** [cbahlai/rqm-template#9]
At some screen resolutions, text on left side of the example box bumps into the midline. I'm scared of breaking the CSS. @auremoser , any suggestions?

![aack](https://cloud.githubusercontent.com/assets/7469265/18008070/b86f05e2-6b74-11e6-9130-5c1c66f48071.jpg)

**Comment by auremoser** [comment]
Hey @cbahlai ! 
Sorry to get back to you so late! I talked to Luke about this at the foundation and we decided this would be the best solution.

Find this section in your style-dev.less file:

```
 .example {
  display: flex;
  margin: 35px 0 0px 0;
  background: @primary-lightest;
  color: @primary-darkest;
  padding: 30px 22px;

  .title {
    flex: 1;
    color: inherit;
    padding-right: 20px; // add this line
```

I have added it and pushed the change to your template (let me know if it works). I hope that helps, give you a little more breathing room in the example sections:
![screen shot 2016-09-16 at 2 26 16 pm](https://cloud.githubusercontent.com/assets/1559703/18596663/55e73656-7c18-11e6-8773-de4e30d3a023.png)
